config/govuk_index/migrated_formats.yaml

```diff
@@ -52,6 +52,8 @@ migrated:
 - hmrc_manual_section
 - manual
 - manual_section
+- minister # from Whitehall, should be `ministerial_role`
```

Comment:
This is a bit of a hack to avoid duplicate results because Whitehall sends these as document_type of `minister` instead of `ministerial_role`.